src/shared/components/cells/CellContext.tsx

```diff
@@ -196,11 +234,14 @@ const CellContext: FC<Props> = ({
   )
 }
 
+const mstp = (state: AppState) => ({
+  variables: getAllVariables(state),
```

Comment:
there's a code smell here that you have to pull variables in just to pass them back down to the cache like that

Reply:
Ya, I think the query cache makes that optional somewhere in one of the caches. We can probably do the same thing here

Reply:
Gonna keep this for now, since we need access to all the variables and @Asalem1 approves